src/components/EventList.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Flex, View } from '@adobe/react-spectrum';
+
+interface EventListProps {
+    events: {
+        activity: { title: string },
+        campaign: { title: string },
+        end_time: string,
+        id: number,
+        location: { title: string },
+        start_time: string,
+        title: string
+    }[],
+    org: {
+        id: number,
+        title: string
+    }
+}
+
+const EventList = ({ events, org } : EventListProps) : JSX.Element => {
+    return (
+        <>
+            <Flex data-test='event-list' direction='row' gap='100' wrap>
+                { events.map((e) => (
+                    <Flex data-test='event' direction='column' margin='size-200' key={ e.id }>
+                        { e.title ? <View data-test='event-title'>{ e.title }</View>
+                            : <View data-test='event-activity-title'>{ e.activity.title }</View> 
```

Comment:
I see no reason to have two separate views here. The only reason I can think of is related to the test, and that's [smelly code](https://en.wikipedia.org/wiki/Code_smell). I've explained in a comment on the test itself how it can be better in a way that makes this redundant.

Instead, we should have just one element, and dynamically switch out the string that it's displaying, not switch out the entire element.

Reply:
I agree, feels a bit smelly.